toolkit/scss/shared_placeholders/_visuallyhidden.scss

```diff
@@ -0,0 +1,4 @@
+%visuallyhidden {
+  position: absolute;
+  left: -9999em;
```

Comment:
Is there a reason for doing this rather than `display: none;`, which seems a lot less hacky.  What if the question is really long?  The tail end of it might end up on the screen if it's enough to fill 10000 pixels.  

If we have to do this hacky positioning out of screen for some reason, out on the right-hand side seems less risky? Anything over 4000 would do it at today's resolutions.